peekingduck/pipeline/nodes/model/hrnetv1/hrnet_model.py

```diff
@@ -0,0 +1,70 @@
+"""
+Copyright 2021 AI Singapore
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import logging
+from typing import Any, Dict, Tuple
+
+import numpy as np
+from peekingduck.weights_utils import checker, downloader
+from peekingduck.pipeline.nodes.model.hrnetv1.hrnet_files.detector import Detector
+
+
+class HRNetModel:
+    """HRNet model to detect poses from detected bboxes
+    """
+
+    def __init__(self, config: Dict[str, Any]) -> None:
+        super().__init__()
+
+        self.logger = logging.getLogger(__name__)
+
+        # check for hrnet weights, if none then download into weights folder
+        if not checker.has_weights(config['root'], config['weights_dir']):
+            print('---no hrnet weights detected. proceeding to download...---')
+            downloader.download_weights(config['root'], config['blob_file'])
+            print('---hrnet weights download complete.---')
+
+        self.detector = Detector(config)
+        self.threshold_score = config['score_threshold']
```

Comment:
Why do we need a `self.threshold_score` as well as a function to get the threshold_score?

Reply:
the function is just to have a getter function so that would not have the too-few-public-methods pylint error. Can remove the getter function and disable pylint instead.

Let me know which you prefer

Reply:
Let's go with the disable pylint approach. Thanks boss

Reply:
okay can no prob